packages/platforms/browser/lib/auto-instrumentation/resource-load-plugin.ts

```diff
@@ -54,9 +54,22 @@ export class ResourceLoadPlugin implements Plugin<BrowserConfiguration> {
         const parentContext = this.spanContextStorage.first
 
         if (parentContext) {
-          const url = new URL(entry.name)
-          url.search = ''
-          const name = url.href
+          const networkRequestInfo = configuration.networkRequestCallback({ url: entry.name, type: entry.initiatorType })
+
+          if (!networkRequestInfo) return
+
+          if (typeof networkRequestInfo.url !== 'string') {
+            configuration.logger.warn(`expected url to be a string following network request callback, got ${typeof networkRequestInfo.url}`); return
```

Comment:
Can you put these returns on their own lines so they're harder to miss?

Reply:
I did put it there actually, but the prettier plugin seemed to have other ideas - I'll try again.